TROUBLESHOOTING.md

```diff
@@ -0,0 +1,192 @@
+# Troubleshooting Common Issues
+
+## Using this Document
```

Comment:
Before this, it should make sure it is really obvious to update `npm`. WIth a link to the part farther down.